conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
```

Comment:
README is missing. For repos that are built using auto tools, please include an instruction to run `make distclean` (to clean up files from previous builds) before running `/opt/bin/build_setup.sh`

Reply:
As per discussion I will address `make distclean` part in presentation and not modifying README for 3rd party libraries